src/algebra/module/projective.lean

```diff
@@ -131,6 +138,26 @@ begin
     simp },
 end
 
+variables {Q : Type (max u v)} [add_comm_monoid Q] [module R Q]
+
+instance [hP : projective R P] [hQ : projective R Q] : projective R (P × Q) :=
+begin
+  rw module.projective_def',
+  cases hP.out with sP hsP,
+  cases hQ.out with sQ hsQ,
+  use coprod (lmap_domain R R (inl R P Q)) (lmap_domain R R (inr R P Q)) ∘ₗ sP.prod_map sQ,
+  ext;
+  simp only [coe_inl, coe_inr, coe_comp, function.comp_app, prod_map_apply, map_zero, coprod_apply,
+  lmap_domain_apply, map_domain_zero, add_zero, zero_add, id_comp,
+  total_map_domain R _ (prod.mk.inj_right (0 : Q)),
+  total_map_domain R _ (prod.mk.inj_left (0 : P))],
+
```

Comment:
```suggestion
  ext;
    simp only [coe_inl, coe_inr, coe_comp, function.comp_app, prod_map_apply, map_zero, coprod_apply,
      lmap_domain_apply, map_domain_zero, add_zero, zero_add, id_comp,
      total_map_domain R _ (prod.mk.inj_right (0 : Q)),
      total_map_domain R _ (prod.mk.inj_left (0 : P))],
```
we generally indent if a single tactic takes more than 1 line (and note that `t; s` counts as a single tactic) and generally don't have empty lines in a proof.

After incorporating this change, feel free to merge (make sure the line length style linter is happy though).

bors d+